CDP4Composition/CommonView/AutoGenRows/ArrayParameterTypeRowViewModel.cs

```diff
@@ -101,9 +121,8 @@ protected override void ObjectChangeEventHandler(ObjectChangedEvent objectChange
         /// </summary>
         private void UpdateProperties()
         {
-            this.ModifiedOn = this.Thing.ModifiedOn;
```

Comment:
Are you sure that ModifiedOn should be removed from the code?

Reply:
yes, this has  been moved to RowViewModelBase as it is a property of Thing